install/install.go

```diff
@@ -283,7 +330,35 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 		if err != nil {
 			return err
 		}
-		fmt.Printf("KubeArmor manifest file saved to \033[1m%s\033[0m\n", f.Name())
+		s3:=f.Name()
+		progbar.filled=48
+		progbar.unfilled=0
+		progbar.done=(float64(progbar.filled)/progbar.size)*float64(100)
+		s:=""
+		for i:=0;i<progbar.filled;i++{
+			s=s+progbar.fill
+		}
+		progbar.bar=s
+		printMessage("🤩  KubeArmor manifest file saved to \033[1m"+s3+"\033[0m",false)
+
+	}
+	fmt.Printf("Checking if Karmor containers are running ...")
+	rcursor:=[4]string{"−","\\","|","/"}
+	rcursorcount:=0
+	for {
+		time.Sleep(300*time.Millisecond)
+		pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app",FieldSelector: "status.phase!=Running"})
+		podno:=len(pods.Items)
+		fmt.Printf("\rKARMOR pods left to run : %d ... %s               ",podno,rcursor[rcursorcount])
+		rcursorcount++
+		if rcursorcount == 4 {
+			rcursorcount=0
+		}
+		if podno == 0 {
+			fmt.Printf("\r🥳  DONE CHECKING , ALL SERVICES ARE RUNNING ...                 \n")
```

Comment:
All caps doesn't seems nice.
fmt.Printf("\r🥳  Done Checking. All Services are running! \n")

Reply:
I really like the emoji used for this :-)
But yeah, lets not use all caps.

Reply:
okay

Reply:
the rcursor change is not working will revert back to the for loop